books/models.py

```diff
@@ -74,6 +74,19 @@ def return_to_library(self, user, library):
 
         send_waitlist_book_available_notification.delay(borrowed_copy.pk)
 
+    def report_as_missing(self, library):
+        book_copy = self.__get_copy(library=library)
+        book_copy.missing = True
+        book_copy.save()
+
+    def was_found(self, library):
+        book_copy = self.__get_copy(library=library)
+        book_copy.missing = False
+        book_copy.save()
```

Comment:
I thought about creating a single method called `report_missing(self, library, missing)` so some parts of the code is not duplicated in two different methods.

I just felt that creating a new method to warn a book was found  sounded more semantic